### ISSUE TITLE
Steel Dust Recipe missing

### ISSUE THREAD

**Issue by DaePrice** [AllTheMods/ATM-3#296]
When crushing steel ingots in mekanism crushers, or AA crushers, the output is pulverized iron. Not tested with any other crushers.

**Comment by DaePrice** [comment]
IC2 macerator also outputs something other than steel dust.

**Comment by MrFootman** [comment]
Only Thermal Expansion Pulverizer seems to give the right output (in case you need it)

**Comment by OrdinatorStouff** [comment]
yeah the TE ingots are the ones going to iron dust instead of steel. Need more investigation but this could be something to do with JAOPCA or UniDict perhaps
![image](https://user-images.githubusercontent.com/6283913/33302249-0550d6a4-d3b8-11e7-93da-e5bb62eaff43.png)

**Comment by Yamiji** [comment]
It's UniDict, without that mod Energetic Infuser produces Mekanism's steel dust like it should.

**Comment by kjmaster1** [comment]
PR #521 Changes the AA crusher, Mekanism crusher and IC2 macerator to output thermal foundation steel blend instead of pulverized iron.
